pkg/vat.go

```diff
@@ -162,16 +161,20 @@ func (v Vat) metrics() metricsReporter {
 }
 
 // match the protocol, ignoring namespace
-func matches(c Capability, proto string) bool {
+func matches(c Capability, proto protocol.ID) bool {
```

Comment:
`Host.Peerstore().GetProtocols(id)` was updated to return `protocol.ID` instead of `string`.